scripts/users-permissions.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+USER="admin:pass"
+HEADER="Content-Type: application/json"
+
+# Alias for curl 
+curl="./bin/curl -u ${USER}"
+
+# Create "Teacher" team
+${curl} -d "name=Teacher" grafana:3000/api/teams
+ 
+# Create "instructor" and "staff" user
+${curl} -d "email=student@example.org&login=staff&password=funfunfun" grafana:3000/api/admin/users
+${curl} -d "email=teacher@example.org&login=instructor&password=funfunfun" grafana:3000/api/admin/users
```

Comment:
You can store the response user ID to use it later and avoid new requests.

Reply:
Also, according to the MySQL fixture, we need an instructor and a student account. not staff/instructor?